news/4.34/jdt.html

```diff
@@ -134,6 +134,22 @@ <h2>Java Views and Dialogs</h2>
 			<img src="images/move-filtersbutton-to-top-other-way.png" alt="old menu">
 		  </p>
       </td>
+	  <tr id="experimental-preference">
+	  	  <td class="title"><strong>Experimental!</strong> Preference added to enable update on runtime behavior</a></td>
+		  <td class="content">
+              <p>
+				This update introduces a new preference setting that allows HiDPI support, enabling monitor-specific UI scaling for improved display clarity. Enabling this preference will also set the <strong>Edge</strong> browser as the default.
+				This check is disabled by default to streamline standard functionality. However, it can be activated if testing of this specific functionality is required. To enable, please configure the appropriate setting as described in the documentation.
```

Comment:
I would keep this information short. Just say that is is disabled by default and that it can be enabled in the preferences, as shown in the screenshot. In my opinion, we should also point out that people are encouraged to try out this feature and give feedback to further improve / finalize it.